### PR TITLE
Add ADS Valve documentation

### PR DESCRIPTION
**Pull request by mrpasztoradam**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125619
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced support for "Valve" devices in the ADS integration, expanding automation capabilities.
	- Added configuration examples for setting up the valve platform, including new parameters for enhanced user customization.

### CODE REVIEW

File: source/_integrations/ads.markdown [hunk 1]
```diff
@@ -27,11 +29,14 @@ The ADS (automation device specification) describes a device-independent and fie
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
+- [Action](#action)
```
Comment (klaasnicolaas) [inline source/_integrations/ads.markdown]: This doesn't belong on the list

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/ads.markdown (hunk 1 above)
Comment (mrpasztoradam) [reply source/_integrations/ads.markdown]: fixed

File: source/_integrations/ads.markdown [hunk 2]
```diff
@@ -253,3 +256,28 @@ device_class:
   description: Sets the [class of the device](/integrations/cover/), changing the device state and icon that is displayed on the frontend.
   type: device_class
 {% endconfiguration %}
+
+## Valve
+
+The `ads` valve platform accesses a boolean variable on the connected ADS device. The variable is identified by its name.
```
Comment (c0ffeeca7) [inline source/_integrations/ads.markdown]:
```suggestion
The `ads` valve entity accesses a boolean variable on the connected ADS device. The variable is identified by its name.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mrpasztoradam :+1:

**Pull request merged**